src/RecipeEntity.php

```diff
@@ -80,6 +80,28 @@ public function generateHTML(): string {
         return $html;
     }
 
+    /**
+     * Generating HTML from recipe properties with delete button for saved recipes.
+     *
+     * @return string HTML code.
+     */
+    public function generateHTMLForSavedRecipes(): string {
```

Comment:
```suggestion
    public function generateHTMLForSavedRecipes(): string
    {
```
gotta keep the brackets like this for methods